client/rpc-core/src/types/call_request.rs

```diff
@@ -43,4 +43,6 @@ pub struct CallRequest {
 	pub data: Option<Bytes>,
 	/// Nonce
 	pub nonce: Option<U256>,
+	/// AccessList
+	pub access_list: Option<Vec<(H160, Vec<H256>)>>,
```

Comment:
Is this in compliance with geth? I think we need to use structs with named fields.
You can see if you can directly use https://docs.rs/ethereum/latest/ethereum/struct.AccessListItem.html

Reply:
Yeah you're right, let me change that.

Reply:
To support this we need to get in https://github.com/rust-blockchain/ethereum/pull/33 first to be compatible with JS libraries.